pkg/kdump/kdump-view.jsx

```diff
@@ -212,14 +212,12 @@ const KdumpSettingsModal = ({ settings, initialTarget, handleSave }) => {
                                        onChange={setServer} isRequired />
                         </FormGroup>
 
-                        <FormGroup fieldId="kdump-settings-ssh-key" label={_("SSH key")}
-                                   helperTextInvalid={validationErrors.sshkey}
-                                   helperTextInvalidIcon={<ExclamationCircleIcon />}
```

Comment:
You dropped the icon, even though the helper component does support it. Accident?

Reply:
The Helper component automatically shows an icon for *all* error variant helpers. Check the hasIcon propoerty in the cockpit-components-form-helper.jsx file. So this rather brings consistently to all our pages, where sometimes we had an icon and sometimes not.

We had an icon i 95% percent of the pages anyway, as FormGroup automatically inserts an icons when validaed="error" as well.

This icon specified here explicity was rather redundant anyway, as it would appear (the same exact icon) even if not passed.

Reply:
Ack, thanks! Resolving.